docs/package.json

```diff
@@ -22,11 +22,11 @@
 		"@types/gtag.js": "^0.0.12",
 		"clipboard-copy": "^4.0.1",
 		"gray-matter": "^4.0.3",
-		"next": "^14.0.4",
+		"next": "^14.2.10",
```

Comment:
Was there a reason why we didn't update this to 15.0.4?

Reply:
All upgrades except Storybook followed our current matchers, e.g. `^` so it's mostly minors. I'm guessing here, but I suspect bumping next to 15 without bumping react to 19 may break in unexpected ways. At some point in the near future we should upgrade major versions and flesh it out in `yourgov`.

Reply:
Makes sense